src/aws_rds_castore.erl

```diff
@@ -3,21 +3,35 @@
 
 -module(aws_rds_castore).
 
--export([file_path/0, ssl_opts/1]).
+-export([file_path/0, file_path/1, ssl_opts/1, ssl_opts/2]).
 
 %% Returns the path to the AWS RDS CA certificate store PEM file.
 file_path() ->
+    file_path(aws).
+
+file_path(aws) ->
+    PrivDir = code:lib_dir(aws_rds_castore, priv),
```

Comment:
I just merged #18 that changes how the path is built. Can you please rebase on `main`?